manifests/deployment.yaml

```diff
@@ -60,8 +60,8 @@ webhooks:
         path: "/validate"
       caBundle: "${CA_BUNDLE}"
     rules:
-      - operations: ["CREATE","UPDATE","DELETE","CONNECT"]
+      - operations: ["*"]
         apiGroups: [""]
         apiVersions: ["v1"]
-        resources: ["pods"]
+        resources: ["*/*"]
```

Comment:
These changes indicate that, when a user performs any type of operation on any type of resources or subresources of k8s, this webhook will get triggered.